packages/devtools_app/lib/src/code_size/code_size_controller.dart

```diff
@@ -123,7 +123,7 @@ class CodeSizeController {
   final _processingNotifier = ValueNotifier<bool>(false);
 
   // TODO(peterdjlee): Spawn an isolate to run parts of this function to
-  //                   prevent the UI from freezing and display a circular 
+  //                   prevent the UI from freezing and display a circular
```

Comment:
OH are you awaiting loadTreeFromJson file where it is called in the UI? this could be why the state was not being updated

Reply:
I tried running it after removing all the asyncs and awaits, but seems like that doesn't fix it. In `loadTreeFromJsonFile`, commenting out everything after `processingNotifier.value = true` makes the loading message pop up, so it seems like the thread is frozen as soon as we enter the function and stays that way until we get out. I'm not sure if this is expected behavior?